src/bilder/images/edxapp_v2/templates/edxapp/mitxonline/common_values.yml.tmpl

```diff
@@ -354,6 +354,7 @@ JWT_EXPIRATION: 30
 LANGUAGE_CODE: en
 LANGUAGE_COOKIE: {{ env "ENVIRONMENT" }}-openedx-language-preference
 LEARNER_PORTAL_URL_ROOT: https://learner-portal-localhost:18000
+LEARNER_HOME_MICROFRONTEND_URL: ''
```

Comment:
I think this should probably be `/dashboard/` which will then trigger the redirect back to the marketing app